core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2443,7 +2469,7 @@ public String toString() {
           .setDefaultValue(29999)
           .setDescription("The port for Alluxio worker's RPC service.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
-          .setScope(Scope.WORKER)
+          .setScope(Scope.ALL)
```

Comment:
why is this `Scope.ALL`?

Reply:
Because this property key is used by Core-common/InstanceConfiguration.validate()